### PR TITLE
Npm publish

### PR DESCRIPTION
**Pull request by G-Ray**
- add source ttf fonts to npmignore
- prepare script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
     "Petr Sloup <petr.sloup@klokantech.com>",
     "Qwant"
   ],
   "scripts": {
-    "build": "node ./generate"
+    "build": "node ./generate",
+    "prepare": "npm run build"
   },
-  "dependencies": {
+  "devDependencies": {
```
Comment (GuillaumeGomez) [inline package.json:13]: Why this change?

Comment (G-Ray) [reply package.json:13]: the package published to npm will provide the dist files. Rebuilding them at install time is unnecessary and should never happend, so I added the source ttf fonts to .npmignore, and `dependencies -> devDependencies`

Comment (G-Ray) [reply package.json:13]: this result in less dependencies installed at install time

File: package.json [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
   "version": "1.0.0",
+  "repository": "github:QwantResearch/fonts",
   "authors": [
     "Petr Sloup <petr.sloup@klokantech.com>",
     "Qwant"
   ],
   "scripts": {
-    "build": "node ./generate"
+    "build": "node ./generate",
+    "prepare": "npm run build"
```
Comment (bbecquet) [inline package.json:11]: Isn't it `prepublishOnly` that you need?

**Review by bbecquet — APPROVED** [review]

**Review by GuillaumeGomez — APPROVED** [review]
